README.md

```diff
@@ -20,6 +18,8 @@ If you want to add a conference, feel encouraged to create a PR which adds the c
 
 | Name                                | Date                       | CfP-Link                           | CfP-Deadline-Date        | Place                               | Aprox. Attendees |
 | ----------------------------------- | -------------------------- | ---------------------------------- | ------------------------ | ----------------------------------- | ---------------- |
+| [Fluttercon USA 2024][11]           | September 19th - 20th, 2024| [Fluttercon USA Cfp][30]           | July 12th, 2024          | New York City, New York             | 700+             |
```

Comment:
Not related to this PR, but there was definitely not 700+ folks attending FlutterCon USA